test/melange-testing-library/README.md

```diff
@@ -0,0 +1,5 @@
+### vendored melange-testing-library
+
+This directory contains a vendored version of the melange-testing-library.
+
+The original repository is https://github.com/melange-community/melange-testing-library
```

Comment:
The PR description mentions

> Vendors ReactTestingLibrary (and does some ad-hoc changes)

are the ad-hoc changes valuable for other users of the bindings? in that case, will they be upstreamed?

Reply:
Not sure, melange-testing-library contains a few binding designs focused on previous versions of BuckleScript where options aren't translated to undefined, or records aren't objects.

I would keep the changes here and evolve this bindings as a playground for the melange-testing-library

Reply:
works, but we should upstream as they get better!